src/components/notebook/cell/Cell.tsx

```diff
@@ -0,0 +1,116 @@
+import { tables } from "@runt/schema";
+import React from "react";
+
+import { AiCell } from "./AiCell.js";
+import { CodeCell } from "./CodeCell.js";
+import { MarkdownCell } from "./MarkdownCell.js";
+import { SqlCell } from "./SqlCell.js";
+
+import { ErrorBoundary } from "react-error-boundary";
+
+type CellType = typeof tables.cells.Type;
+
+interface CellProps {
+  cell: CellType;
```

Comment:
This should probably get named `cellType` as the variable so we (ok a confused AI) never try to use this to pass a whole cell.